### PR TITLE
Adding missing route response parameters to getRoutes()

### PR DESCRIPTION
**Pull request by grunch**
https://github.com/alexbosworth/ln-service/issues/35

### CODE REVIEW

File: lnd/routes_from_query_routes.js [hunk 1]
```diff
@@ -59,12 +67,16 @@ module.exports = ({response}) => {
...
       const totalFeesMsat = new BN(route.total_fees_msat, decBase);
+      const totalAmtMsat = new BN(route.total_amt_msat, decBase);
 
       return {
         fee: totalFeesMsat.div(msatsPerToken).toNumber(),
+        fee_mtokens: totalFeesMsat.toString(),
         timeout: route.total_time_lock,
+        tokens: totalAmtMsat.div(msatsPerToken).toNumber(),
+        mtokens: totalAmtMsat.toString(),
+        hops: route.hops,
```
[the remark this answers is not in the document]
Comment (grunch) [reply lnd/routes_from_query_routes.js]: I don't understand this 'Should hops be spelled out a bit more? I wouldn't really know what to expect to get off the top of my head.' I'm sorry, non english native speaker.

I updated the route from query test

[the remark this answers is not in the document]
Comment (alexbosworth) [reply lnd/routes_from_query_routes.js]: Cool

I mean for hops that it should be commented out a bit to indicate what is returned

```
hops: [{
 id: <Id String>
}]
```
That kind of thing

Normally all the return values are spelled out

[the remark this answers is not in the document]
Comment (grunch) [reply lnd/routes_from_query_routes.js]: oh yeah, of course :+1:

**Comment by grunch** [comment]
haha yeah I was going to start asking about sendtoroute, great improvement :smile:

**Pull request merged**